support-frontend/assets/helpers/tracking/acquisitions.ts

```diff
@@ -111,26 +125,17 @@ function storeReferrerAcquisitionDataInSessionStorage(
 
 const toAcquisitionQueryParameters = (
 	parameters: Array<[string, string]>,
-): AcquisitionQueryParameters =>
-	parameters.reduce((array, item) => {
-		array.push({
-			name: item[0],
-			value: item[1],
-		});
-		return array;
-	}, []);
+): AcquisitionQueryParameters => {
+	return parameters.map(([name, value]) => ({ name, value }));
+};
 
 const participationsToAcquisitionABTest = (
 	participations: Participations,
 ): AcquisitionABTest[] => {
-	const response: AcquisitionABTest[] = [];
-	Object.keys(participations).forEach((participation) => {
-		response.push({
-			name: participation,
-			variant: participations[participation as any],
-		});
-	});
-	return response;
+	return Object.entries(participations).map(([name, variant]) => ({
+		name,
+		variant,
+	}));
```

Comment:
I'm not quite sure the history of these functions, but I think now they're best of being `map`s!

Reply:
💯